lib/linguist/heuristics.yml

```diff
@@ -169,6 +182,15 @@ disambiguations:
     pattern: '^\s*(?:%%|main\s*\(.*?\)\s*->)'
   - language: JavaScript
     pattern: '(?m:\/\/|("|'')use strict\1|export\s+default\s|\/\*.*?\*\/)'
+- extensions: ['.ex']
+  rules:
+  - language: Elixir
+    pattern:
+    - '^\s*@moduledoc\s+'
```

Comment:
There's no need for the trailing quantifier (`\s+`), because the regex need only match a single whitespace character in order to succeed:

~~~suggestion
    - '^\s*@moduledoc\s'
~~~